.github/workflows/update-go-fluentbit-config.yaml

```diff
@@ -0,0 +1,48 @@
+name: Update go-fluentbit-config schema
+
+on:
```

Comment:
I think we want to run this the other way, i.e. when changes are made on the schema side.
We probably want to schedule this to run regularly and then it'll just happen automatically.

Reply:
Ah my bad, I thought this was in the other repo!